nix-mode.el

```diff
@@ -520,6 +520,22 @@ STRING-TYPE type of string based off of Emacs syntax table types"
     (when (> end-of-indentation (point)) (goto-char end-of-indentation)))
   )
 
+;; ffap support
+(defun nix-mode-ffap-nixpkgs-path (str)
+  "Support `ffap' for <nixpkgs> declarations.
+If STR contains nixpkgs, call nix-instantiate to find the
+location of STR."
+  ;; The < > around <nixpkgs> don't get passed to us
+  (when (string-match "nixpkgs" str)
```

Comment:
Hmmm... can this be altered to generally match any path within `<...>`s? It won't always be `<nixpkgs...>` but instead could be almost anything.

Reply:
The `<` and `>` characters don't get passed through to us, unfortunately. So we can't tell if we're inside <> or not.

Reply:
You can keep the brackets with:

```
(push '(nix-mode "--:\\\\${}<>+@-Z_[:alpha:]~*?" "@" "@;.,!:") ffap-string-at-point-mode-alist)
```